### DISCUSSION TITLE
Do the functions of UPPAAL 5.0 include those of UPPAAL tiga?

### DISCUSSION THREAD

**Discussion by anuli1234566777765768** [UPPAALModelChecker/UPPAAL-Meta#275] [Q&A]
UPPAAL tiga version not running on windows? No .exe file?
![Uploading 微信图片_20240531170559.png…]()

**Comment by mikucionisaau** [comment]
The image does not load for me, so I do not see it.

UPPAAL 5 verifier includes TIGA queries, but the concrete simulator cannot play games yet (only simulate).

**Comment by anuli1234566777765768** [comment]
Thank you very much for your reply! May I understand that for the same .xml file, the results of the validator of UPPAAL and the validator of UPPAAL tiga are the same?
Because the UPPAAL tiga version doesn't run on windows because there is no .exe file.（see fig
![Uploading 微信图片_20240531170559.png…]()
） Can you send me a zip file of UPPAAL tiga that can be operated on windows?

**Comment by anuli1234566777765768** [comment]
<img width="646" alt="1" src="https://github.com/UPPAALModelChecker/UPPAAL-Meta/assets/155973896/f1c187ef-1591-4397-a943-77242d24c031">

**Reply by mikucionisaau** [reply]
`uppaal.exe` is just a launcher added in recent years.

For earlier releases just double-click `uppaal.jar` (assuming that Java runtime is installed with `jar` associations).
